[BUG] No version for EXPath Crypto dependency

The existdb-saml EXPath Package (including the latest version  1.6.3) does not declare the version of the EXPath Crypto module that it depends on.

The `expath-pkg.xml` in the XAR file, only contains this:
```xml
<package xmlns="http://expath.org/ns/pkg" name="http://exist-db.org/xquery/exsaml" abbrev="existdb-saml" version="1.6.3" spec="1.0">
    <title>existdb-saml</title>
    <dependency package="http://expath.org/ns/crypto"/>
    <dependency processor="http://exist-db.org" semver-min="3.2.0"/>
</package>
```

The dependency on `http://expath.org/ns/crypto` should declare at least a `semver-min`.

There have been lots of differing version of the EXPath Crypto module (published for eXist-db), including crypto versions: `6.0.1`, `6.0.0`, `5.3.0`, `1.0.0`, `0.7`, `0.6`, `0.5`, `0.3.5`, and `0.3.4`.

The `existdb-saml` module declares that it requires eXist-db version 3.2.0 or later. The version of EXPath Crypto for eXist-db 3.2.0 (i.e. 0.3.5) however does not appear to work with the `existdb-saml` module on eXist-db 6.1.0. See https://exist-db.org/exist/apps/public-repo/packages/crypto?eXist-db-min-version=3.2.0

## Questions
1. Which is the version that is meant to be used by existdb-saml please?
2. Does the semver-min of eXist-db used by the existdb-saml module need to be updated perhaps?

I have also just tested the following which also appears to fail:
* eXist-db 6.1.0-SNAPSHOT
* existdb-saml 1.6.3
* EXPath Crypto 6.0.1

I get the following error:
```
Caused by: org.exist.xquery.XPathException: exerr:ERROR error found while loading module exsaml: Error while loading module xmldb:///db/apps/existdb-saml/content/exsaml.xqm: error found while loading module crypto: Cannot find module class from EXPath repository: org.expath.exist.crypto.ExistExpathCryptoModule
	at org.exist.repo.ExistRepository.getModule(ExistRepository.java:164)
	at org.exist.repo.ExistRepository.resolveJavaModule(ExistRepository.java:140)
	at org.exist.xquery.XQueryContext.resolveInEXPathRepository(XQueryContext.java:509)
	at org.exist.xquery.XQueryContext.importModule(XQueryContext.java:2450)
	at org.exist.xquery.parser.XQueryTreeParser.importDecl(XQueryTreeParser.java:6353)
	at org.exist.xquery.parser.XQueryTreeParser.prolog(XQueryTreeParser.java:5367)
	at org.exist.xquery.parser.XQueryTreeParser.libraryModule(XQueryTreeParser.java:4047)
	at org.exist.xquery.parser.XQueryTreeParser.module(XQueryTreeParser.java:3878)
	at org.exist.xquery.parser.XQueryTreeParser.xpath(XQueryTreeParser.java:3659)
	at org.exist.xquery.XQueryContext.compileModule(XQueryContext.java:2666)
	at org.exist.xquery.XQueryContext.compileOrBorrowModule(XQueryContext.java:2607)
	at org.exist.xquery.XQueryContext.importModuleFromLocation(XQueryContext.java:2536)
	at org.exist.xquery.XQueryContext.importModule(XQueryContext.java:2471)
	at org.exist.xquery.parser.XQueryTreeParser.importDecl(XQueryTreeParser.java:6353)
	at org.exist.xquery.parser.XQueryTreeParser.prolog(XQueryTreeParser.java:5367)
	at org.exist.xquery.parser.XQueryTreeParser.mainModule(XQueryTreeParser.java:4059)
	at org.exist.xquery.parser.XQueryTreeParser.module(XQueryTreeParser.java:4004)
	at org.exist.xquery.parser.XQueryTreeParser.xpath(XQueryTreeParser.java:3659)
	at org.exist.xquery.XQuery.compile(XQuery.java:251)
	at org.exist.xquery.XQuery.compile(XQuery.java:180)
	at org.exist.xquery.XQuery.compile(XQuery.java:138)
	at org.exist.http.urlrewrite.XQueryURLRewrite.runQuery(XQueryURLRewrite.java:671)
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:241)
	... 37 more
Caused by: java.lang.ClassNotFoundException: org.expath.exist.crypto.ExistExpathCryptoModule
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:467)
	at org.exist.repo.ExistRepository.getModule(ExistRepository.java:154)
	... 59 more
```

I believe the additional error in this comment may instead be related to https://github.com/eXist-db/exist/issues/4901

--- a/README.md
+++ b/README.md
@@ -1,7 +1,6 @@
 # existdb-saml - SAML v2.0 Implementation in XQuery
 
-**NOTE** You will need **eXist v4.4 or higher** to run this code (a required
-compression function has been added in that release).
+**NOTE** You will need **eXist-db version 6.0.1 or higher** to run this code.
 
 ## Overview
 

--- a/expath-pkg.xml.tmpl
+++ b/expath-pkg.xml.tmpl
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://expath.org/ns/pkg" name="@url@" abbrev="@name@" version="@version@" spec="1.0">
     <title>@name@</title>
-    <dependency package="http://expath.org/ns/crypto"/>
-    <dependency processor="http://exist-db.org" semver-min="3.2.0"/>
+    <dependency package="http://expath.org/ns/crypto" semver-min="6.0.1"/>
+    <dependency processor="http://exist-db.org" semver-min="6.0.1"/>
 </package>

